.vscode/launch.json

```diff
@@ -1,6 +1,24 @@
 {
   "version": "0.2.0",
   "configurations": [
+  {
+    "args": [
+      "--extensionDevelopmentPath=${workspaceFolder}"
+    ],
+    "name": "Launch Extension",
+    "outFiles": [
+      "${workspaceFolder}/out/**/*.js"
+    ],
+    "preLaunchTask": "npm",
+    "request": "launch",
+    "type": "pwa-extensionHost"
+  },
+    {
+      "command": "npm start",
+      "name": "Run npm start",
+      "request": "launch",
+      "type": "node-terminal"
+    },
```

Comment:
This change is not related to #6 or #9. I suggest don't make changes to `.vscode/launch.json`